.circleci/config.yml

```diff
@@ -219,6 +219,12 @@ workflows:
     jobs:
       - commitlint/lint:
           target-branch: develop
+          filters:
```

Comment:
Went with `filters` rather than a `when` condition on the workflow because this followed the pattern of other jobs in this file. Also for consistency, I did not include `release` branches since no other jobs in this file referenced a release branch. Seems like we would want to make multiple changes to this config if we want to support `release` branches.